tutorial/train_horizon_net/README.md

```diff
@@ -4,18 +4,26 @@ In this tutorial, we will walk you through how to train your own model using pil
 
```

Comment:
@chinhsuanwu Please, change the tittle into a more meaningful one. eg. Tutorial 1: Training with HN using GT labels